.circleci/scripts/chrome-install.sh

```diff
@@ -4,11 +4,11 @@ set -e
 set -u
 set -o pipefail
 
-CHROME_VERSION='79.0.3945.117-1'
+CHROME_VERSION='87.0.4280.88-1'
 CHROME_BINARY="google-chrome-stable_${CHROME_VERSION}_amd64.deb"
 CHROME_BINARY_URL="http://mirror.cs.uchicago.edu/google-chrome/pool/main/g/google-chrome-stable/${CHROME_BINARY}"
```

Comment:
We only switched to this mirror because google took v79 down. We can switch back to google if we're updating it again.

The link is
```suggestion
CHROME_BINARY_URL="https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/${CHROME_BINARY}"
```

If we're sticking with the mirror, we should at least verify that the hash matches the one from Google. But I expect it'll be faster to download this from Google than from uchicago.edu anyway.